lib/active_shipping/location.rb

```diff
@@ -161,5 +164,17 @@ def address2_and_3
     def ==(other)
       to_hash == other.to_hash
     end
+
+    def first_name
+      @name.split(' ').first
+    end
+
+    def last_name
+      @name.split(' ').last
```

Comment:
Do we not have any better way of doing this? This breaks in plenty of cases. `Von Bargen` `Van Gelder`